tests/ast/data/DataFrame.join_table_function.test.DISABLED

```diff
@@ -0,0 +1,40 @@
+## TEST CASE
```

Comment:
Should this test be enabled with this pr?

Reply:
+1

Reply:
Still having trouble with local tests. I'm working on finding a way to test this with integration tests.

Reply:
Will continue working on enabling these tests in some form (integration test) separately from this PR.

Reply:
Okay, work on separate from this pr.